cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/SystemAppModule.java

```diff
@@ -61,9 +60,15 @@
  */
 public class SystemAppModule extends AbstractModule {
 
+  private final CConfiguration cConf;
+
+  SystemAppModule(CConfiguration cConf) {
+    this.cConf = cConf;
+  }
+
   @Override
   protected void configure() {
-    install(new ConfigModule());
+    bind(CConfiguration.class).toInstance(cConf);
```

Comment:
Should use the `ConfigModule` with the cConf passed in. Also, don't use `install`. Provides all the modules to the `Guice.createInjector` call. See `ConfiguratorTask` for example.

Reply:
Done.